equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/util/ProfileAcceptedTypesUtil.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.justice.laa.crime.equinity.historicaldata.util;
+
+import jakarta.annotation.Nullable;
+import lombok.experimental.UtilityClass;
+import uk.gov.justice.laa.crime.equinity.historicaldata.exception.UnauthorizedUserProfileException;
+
+@UtilityClass
+public class ProfileAcceptedTypesUtil {
+    public static void checkTypeIsAcceptedByProfile(Integer requiredType, @Nullable String types) throws UnauthorizedUserProfileException {
+        if (types == null) return;
```

Comment:
formatting
```
if (types == null) 
    return null;
```

Reply:
it is void. It does not return anything. `return;` is accepted